aptos-move/framework/aptos-framework/sources/Genesis.move

```diff
@@ -172,7 +192,7 @@ module AptosFramework::Genesis {
             Coin::register<TestCoin>(&owner_account);
             Coin::transfer<TestCoin>(&core_resource_account, *owner, amount);
             Stake::add_stake(&owner_account, amount);
-            Stake::join_validator_set(&owner_account, *owner);
+            Stake::join_validator_set_internal(&owner_account, *owner);
```

Comment:
what does `join_validator_set_internal` mean?

Reply:
This does not have the check that validators can join post genesis or not and is only to be called during Genesis (friend-only). The public version that can be called by anyone has that check.